src/planscape/plan/views.py

```diff
@@ -315,6 +318,37 @@ def get_project(request: HttpRequest) -> HttpResponse:
         return HttpResponseBadRequest("Ill-formed request: " + str(e))
 
 
+@csrf_exempt
+def delete_projects(request: HttpRequest) -> HttpResponse:
+    try:
+        # Check that the user is logged in.
+        owner = _get_user(request)
+
+        body = json.loads(request.body)
+        project_ids = body.get('project_ids', None)
+        if project_ids is None or not (isinstance(project_ids, list)):
+            raise ValueError("Must specify project_ids as a list")
+
+        projects = [Project.objects.get(id=project_id) for project_id in project_ids]
+        
+        # Check that the user owns the projects
+        for project in projects:
+            if project.owner != owner:
+                raise ValueError(
+                    "You do not have permission to delete one or more of these projects.")
+        
+        for project in projects:
+            project.delete()
+        
+        response_data = project_ids
```

Comment:
let's only return the ids of the projects we were able to delete

Reply:
Since we won't do partial deletes, that is the current behavior :)